### DISCUSSION TITLE
#功能支持讨论

### DISCUSSION THREAD

**Discussion by slideslide** [GaiZhenbiao/ChuanhuChatGPT#995] [General]
话说川虎哥有支持[function-calling](https://platform.openai.com/docs/guides/function-calling/function-calling)等更多调用方式的想法吗？ @GaiZhenbiao

**Comment by GaiZhenbiao** [comment]
这个不是用来直接用的，而是用的Agent集成等。

**Reply by slideslide** [reply]
就是说有往更多agent方向上做的想法吗？现在只有一个川虎助手（勉强算是？）
